src/mir/index.md

```diff
@@ -256,3 +258,12 @@ but [you can read about those below](#promoted)).
 [mir]: https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/mir/index.html
 [mirmanip]: https://doc.rust-lang.org/nightly/nightly-rustc/rustc_mir/index.html
 [newtype'd]: ../appendix/glossary.html#newtype
+[basicblocks](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/mir/struct.Body.html#structfield.basic_blocks)
```

Comment:
This should be:
```markdown
[basicblocks]: https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/mir/struct.Body.html#structfield.basic_blocks
```

Reply:
I fixed this at https://github.com/rust-lang/rustc-dev-guide/pull/984, and reflowed the page.